### ISSUE TITLE
[bug] pajbot banphrase check false negative with /me messages

### ISSUE THREAD

**Issue by noiredayz** [hotbear1110/botbear#33]
I noticed that your bot is getting timed out by the epic "spelling it" banphrase on the bruh test command in hobos channel when the supplied parameter is just an n letter.
If you check that message, it's banphrased:
```json
{
    "banned": true,
    "banphrase_data": {
        "case_sensitive": true,
        "id": 1347,
        "length": 30,
        "name": "Spelling it",
        "operator": "regex",
        "permanent": false,
        "phrase": "^[nN]\\b(?! OMEGALUL(iguess| PE)?$)",
        "remove_accents": true,
        "sub_immunity": false
    },
    "input_message": "n did not pass the bruh test"
}

```
However when you leave the /me inside, it it not marked as banned by the API, I guess this is the message your bot checks:
```json
{
    "banned": false,
    "input_message": "/me n did not pass the bruh test"
}
```
But since pajbot checks just the actual message sent to the chat, which doesn't contain the command it times out your bot for 30 secs.
Suggestion: you could strip the leading "/me " (incl. the space) from the message sent to the API so regexes and "startswith" type banphrases won't provide false negatives then a timeout.

**Comment by hotbear1110** [comment]
LuL oh, thanks for the info, Will fix that

**Comment by hotbear1110** [comment]
fixed in f484204
